### ISSUE TITLE
Native WordPress custom fields are added twice

### ISSUE THREAD

**Issue by dimitrismitsis** [WordPress/gutenberg#11976]
Adding a custom field in a post using the native fields in WordPress prior to publishing, results in the custom fields being added twice.

Steps to reproduce this:
1. Go to 'wp-admin/post-new.php'
2. Through the options in the top-right, enable the Custom Field advanced panel
3. Add a custom field with name and value and hit "Add Custom Field"
4. Next, add a title and some content in a Paragraph block 
5. Publish the post
6. Reload the page

**Expected behavior**
The custom field should appear only once in the page. 
Instead, it appears twice.

This is reproducible only if you add the Custom field prior to the title and content.

**Screenshot**
<img width="1008" alt="screenshot 2018-11-16 at 14 47 25" src="https://user-images.githubusercontent.com/15674753/48622161-8e8d3400-e9ae-11e8-9e68-255587086a78.png">

**Desktop (please complete the following information):**
 - OS: macOS Mojave 10.14.1
 - Browser Chrome
 - Version 70.0.3538.102 (Official Build) (64-bit)

Using WordPress 5.0-beta5-43907
Gutenberg: Version 4.4.0

**Comment by pbrocks** [comment]
Confirming the issue with a screencast showing the relevance of the order of operations:

![confirming-custom-field-issue](https://user-images.githubusercontent.com/5721313/48639347-4e738480-e9a1-11e8-9817-3b32a3646259.gif)

**Comment by garrett-eclipse** [comment]
This was flagged in Core Trac here - https://core.trac.wordpress.org/ticket/47576

Reviewing I found it only affects the block editor and not the classic editor. As well I was able to reproduce both when the fields are added before the title/content as well as after they are added. And it occurs both when Save Draft or Publish actions are used.

**Comment by donmhico** [comment]
+1 Issue still persist in **WP 5.3-alpha-45282-src**.

**Comment by wdslatter** [comment]
Has this been resolved? From what I can tell it persists in 5.7.2

**Comment by carolinan** [comment]
Still reproducible in 6.0.1 with and without Gutenberg 13.7.0-rc.1

**Comment by jordesign** [comment]
I can no longer recreate this in WP 6.2.2 - so I'm going to close the issue. If it is still lurking somewhere - please re-open the issue with further testing details.

**Comment by jsit** [comment]
I'm still seeing this issue in WP 6.7.1. This happens with standard post types, with the Twenty Twenty-Five theme, and with no plugins activated.

@jordesign @carolinan

**Comment by carolinan** [comment]
Please create a new issue with complete and detailed reproduction steps.
